src/betterproto/plugin/models.py

```diff
@@ -577,11 +594,10 @@ def __post_init__(self) -> None:
         # Add method to service
         self.parent.methods.append(self)
 
-        # Check for Optional import
+        # Check for imports
         if self.py_input_message:
             for f in self.py_input_message.fields:
-                if f.default_value_string == "None":
-                    self.output_file.typing_imports.add("Optional")
+                f.add_imports_to(self.output_file)
```

Comment:
Do I understand correctly that this line is the actual fix?

Reply:
Yes, along with the other changes.

I've deleted adding "Optional" import two lines above because [there already is the code to add it](https://github.com/danielgtaylor/python-betterproto/pull/183/files#diff-690d271c05506485f2af01f6bc49011358850244a4b65c3f79e21a4d27444c8aR364-R365)) in the `FieldCompiler` class:
```python
if "Optional[" in annotation:
    imports.add("Optional")
```